.github/workflows/deploy.yml

```diff
@@ -106,13 +106,20 @@ jobs:
           # For posting a rich message using Block Kit
           payload: |
             {
-              "text": "GitHub Action build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}",
-              "blocks": [
+              "text": ":alarm: Build on `main` failed! :alarm:\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}",
+             	"blocks": [
                 {
                   "type": "section",
                   "text": {
                     "type": "mrkdwn",
-                    "text": "GitHub Action build result: ${{ job.status }}\n${{ github.event.pull_request.html_url || github.event.head_commit.url }}"
+                    "text": ":alarm: Build on `main` failed! :alarm:"
+                  }
+                },
+                {
+                  "type": "section",
+                  "text": {
+                    "type": "mrkdwn",
+                    "text": "Please check the related commit: ${{ github.event.pull_request.html_url || github.event.head_commit.url }}\n \\cc @zeebe-medic"
```

Comment:
:wrench: Since this workflow doesn't run on pull requests, only on push, it doesn't have access to `github.event.pull_request.html_url` so I think we can remove that part. `github.event.head_commit.url` should be available though :+1: